### ISSUE TITLE
Warn about excessive func args

### ISSUE THREAD

**Issue by quasilyte** [VKCOM/noverify#142]
## Code Example

```php
stat($filename, $flags);
```

## Actual Behavior

No warnings.

## Expected Behavior

Warning about extra (excessive) argument passed to `stat` (or any other) function.

Seems reasonable for functions that don't use `func_get_args` and are not variadic.

**Comment by YuriyNasretdinov** [comment]
Yeah, but you have to make sure that all the internal (and extension) functions are annotated as having variadic arguments. I think they do but it is worth checking that `printf` for example is defined as having variadic arguments.
